library/geonetwork

```diff
@@ -0,0 +1,28 @@
+# this file is generated via https://github.com/geonetwork/docker-geonetwork/blob/c3eba2643bc57c9679ad0cfce1ab5bd7e33fea2d/generate-stackbrew-library.sh
+
+Maintainers: Joana Simoes <joana.simoes@geocat.net> (@doublebyte1)
+GitRepo: https://github.com/geonetwork/docker-geonetwork
+
+Tags: 3.0.4
+GitCommit: b65e0df997508b59559e7f5e1be06ee12a8f8567
+Directory: 3.0.4
+
+Tags: 3.0.4-postgres
+GitCommit: 35a1391f8bd9c8d607cc9ac9fc1fa753160df79c
+Directory: 3.0.4/postgres
+
+Tags: 3.0.5, 3, latest
```

Comment:
Related to your comment, I've been reviewing the geonetwork versions and discussing with the other developers and we think it makes more sense to keep just two versions; the tip of the branch 3.0.x and the tip of the branch 3.2.x (tagged as latest). This is reflected on the latest version of the [library file](https://github.com/GeoCat/official-images/blob/96b9ccb845029612daea943ad3d10acc45a22198/library/geonetwork).